commands/koth_quotes.py

```diff
@@ -0,0 +1,50 @@
+from commands.base import BaseCommand
+from random import randint
+
+class KOTHQuotes(BaseCommand):
+    trigger = 'quote'
+    description = 'HuskieBot will say a quote from the best show ever made King of the Hill'
+
+    CHOICES = [
```

Comment:
consider switching from "" to '' to remove escape charcters

Reply:
This was fixed in 8175b47 by just removing the inner quotes